src/index.ts

```diff
@@ -37,6 +38,7 @@ async function init() {
         app.use(urlencoded({ extended: true }));
 
         app.get("/docs", cors(), serveOpenAPIDocs(natsConnection));
+        app.get("/health-check", cors(), healthCheck(natsConnection));
```

Comment:
1. do we need to distinguish between readiness and liveness? I'm curious about our overall strategy for determining if a container should be restart and receive traffic
2. do we need cors or can we remove it?